src/lib/Sympa/Config/Schema.pm

```diff
@@ -1353,7 +1353,7 @@ our %pinfo = (
             'Header fields to be removed before message distribution',
         gettext_comment =>
             "The removal happens after Sympa's own header fields are added; therefore, it is a convenient way to remove Sympa's own header fields (like \"X-Loop:\" or \"X-no-archive:\") if you wish.",
-        format     => '\S+',
+        format_s   => '$header_field_name(:.+)?',
```

Comment:
The previous regex prevented space, but not this one. Is this normal?

Reply:
The code in [TransformOutgoing.pm](https://github.com/sympa-community/sympa/blob/55598aebb2c0ba4b8b5be733563cad89973cf2fc/src/lib/Sympa/Spindle/TransformOutgoing.pm#L145-L147) looks assuming a parameter value such as `Header-Field-Name: value`.  I think the white space(s) between colon and value is optional anyway.

Reply:
:+1: